compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -174,6 +174,9 @@ int entry(int argc, char **argv)
   {
     auto graph = module->graph(idx);
 
+    // quantize the graph
+    optimizer.quantize(graph);
```

Comment:
I thought you would call quantize after optimize :)

Reply:
`optimize` does type inference. If we do quantize before optimize, the type inference becomes wrong. @mhs4670go  pointed this out. Thanks.

Reply:
Ah... intersting :)